testbench/scripts/get-dishes.sh

```diff
@@ -10,6 +10,6 @@ if [ "#$ID_TOKEN" == "#" ] || [ "#$SESSION_TOKEN" == "#" ] ; then
 fi
 
 curl -i -X GET \
-    -H "x-api-key: test" \
+    -H "x-api-key: $API_KEY" \
     -H "session-token: $SESSION_TOKEN" \
-    http://localhost:8080/api/dish
\ No newline at end of file
+    "http://localhost:8080/api/dish/?all=$ALL_DISHES&borrowed=$BORROWED&transaction=$WITH_TRANSACTIONS"
```

Comment:
nit:
The extra `/` is not required after dish
```suggestion
    "http://localhost:8080/api/dish?all=$ALL_DISHES&borrowed=$BORROWED&transaction=$WITH_TRANSACTIONS"
```